client/OPUNetGameSelectWnd.h

```diff
@@ -45,7 +45,7 @@ class OPUNetGameSelectWnd : public IDlgWnd
 	void InitializeServerAddressComboBox();
 	void InitializeGameSessionsListView();
 	void InitializeNetTransportLayer();
-	bool InitGurManager();
+	bool InitializeGuaranteedSendLayerManager();
```

Comment:
Small note, but debug strings from the original game referred to it as the "GurManager". Hence the reason for the name here. We don't need to match exactly though.